light-clients/ics10-grandpa-cw/src/context.rs

```diff
@@ -0,0 +1,40 @@
+use std::{cell::RefCell, fmt, fmt::Debug, marker::PhantomData, rc::Rc};
+
+use cosmwasm_std::{DepsMut, Storage};
+
+use grandpa_light_client_primitives::HostFunctions;
+use ibc::core::ics26_routing::context::ReaderContext;
+use ics10_grandpa::client_message::RelayChainHeader;
+
+pub struct Context<'a, H> {
+	pub deps: Rc<RefCell<DepsMut<'a>>>,
```

Comment:
this `Rc<..>` magic is to be able to make it clonable right?

Reply:
Yes, but tbh I'm not sure if we'll be using the storage in the such way at all. Maybe just an owned version is enough